src/database/schema/ChannelListSchema.ts

```diff
@@ -1,3 +1,4 @@
+import moment from 'moment';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `moment` library is a good addition for handling date and time related operations. However, it's worth noting that the `moment` library is quite large and can increase your bundle size significantly. If you're only using it for simple operations like sorting dates, consider using smaller libraries like `date-fns` or `dayjs`, or even native JavaScript Date methods if possible.

```diff
- import moment from 'moment';
+ import { compareDesc } from 'date-fns';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->